script/runc.spec

```diff
@@ -0,0 +1,64 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _sbindir /usr/local/sbin
+%global _mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
+
+Name: runc
+Version: %{_version}
+Release: %{_release}
+Summary: A tool for managing OCI containers
+
+License: ASL 2.0
+Source: %{name}-%{version}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: gcc-go
+BuildRequires: glibc
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+BuildRequires: libapparmor-devel
+BuildRequires: go-md2man
```

Comment:
@hqhq @moypray if it's meant to be a guideline file, you might just wanna grab the fedora spec file and adapt it to include other distros. There's a lot of stuff in there already as per fedora's golang packaging draft (including -devel files etc.) which could be useful elsewhere too. See: http://pkgs.fedoraproject.org/cgit/rpms/runc.git/tree/runc.spec

RE: Makefile `rpm` target, I feel it goes in a loop. Meaning, `make rpm` calls `rpmbuild -ba` which then uses the same Makefile but this time inside the untarred sources in the build dir. I guess you could get straight to `rpmbuild -ba` without needing a `make rpm`.

RE: Source0, perhaps you might want to list it as mentioned in  http://pkgs.fedoraproject.org/cgit/rpms/runc.git/tree/runc.spec#n40 , so the user could just `spectool -g runc.spec` to conveniently fetch the tarball.

Anyway, just my 2 cents. If people want to move ahead with this, please don't let me slow you down.